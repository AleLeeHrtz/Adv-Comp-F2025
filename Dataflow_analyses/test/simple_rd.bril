@main {
  x: int = const 1;
  y: int = const 2;
  z: int = add x y;
  x: int = const 3;
  w: int = mul x z;
}
