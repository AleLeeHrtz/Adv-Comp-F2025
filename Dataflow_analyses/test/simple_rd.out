=== Reaching Definitions Analysis ===
entry:
  in:  âˆ…
  out: ('entry', 1, 'y'), ('entry', 2, 'z'), ('entry', 3, 'x'), ('entry', 4, 'w')
